data/starts.txt

```diff
@@ -67,19 +68,19 @@ start "hai space"
 	planet "Greenwater"
 	date 16 11 3013
 	to reveal
-		has "Ask the Hai about the Unfettered: offered"
+		has "Ask the Hai about the Unfettered"
 	on reveal
 		name "Hai Origins"
 		description `You grew up on Greenwater, a Hai world with a rich and mixed culture thanks to its climate. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. To reach that goal, you worked hard at a travel company, recommending locations in human space.`
 		description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
 		system "Fah Soom"
 		planet "Greenwater"
-		date 16 11 3013
-		credits 1431000
-		debt 1431000
+		date Tue, 16 Nov 3013
```

Comment:
When this is loaded, it simply puts the token after "date" into a string.
What you have here will result in the "date" that is displayed when the start is in this state being "Tue,".
```suggestion
		date "Tue, 16 Nov 3013"
```